Code/Framework/AzCore/AzCore/IO/Streamer/Streamer.cpp

```diff
@@ -254,52 +254,78 @@ namespace AZ::IO
 
     void Streamer::RecordStatistics()
     {
+        // create a buffer to reuse for wstring conversions in this loop calling AZStd::to_wstring:
+        constexpr const size_t MaxStatNameLength = 256;
```

Comment:
nit. `constexpr const` is redundant. Due to lambda capture, maybe in this case only `const` should be used.